docs/faq.rst

```diff
@@ -0,0 +1,114 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    For a high level view of the subject, please see the :ref:`further reading <further_reading>`
+    section, where we provide links to a variety of books and papers.
+    In our :ref:`user guide <user_guide>` we also have links to the papers describing
+    Fairlearn's mitigation algorithms.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
+
+    To elaborate, information is often redundantly encoded in datasets, and machine learning
+    algorithms *will* uncover these links (it is what they are designed to do).
+    For example, in the US, the ZIP code where a person lives is well correlated with their
+    race.
+    Even if the model is not provided with race as a feature, the model will pick up on it
+    implicitly via the ZIP code (and other features).
+    Worse, without having the race available in the dataset, it is not possible to determine
+    whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+The model is unfair because the data are biased. Isn't it better to get better data?
+    If a model is unfair due to biased data (note that this is not the only reason a model
+    may be unfair), then obtaining unbiased data is definitely the best solution.
+    However, doing so might not always be practical.
+    In these cases, the mitigation techniques in Fairlearn and similar packages can redistribute
```

Comment:
I don't like the optics of redistribution here, because it encourages zero-sum thinking about allocation harms. I would instead say something like. "There are often many machine learning models that achieve similar levels of accuracy or other performance metrics, but that dramatically differ in how they affect different subgroups. Mitigation algorithms seek to improve the fairness metrics without strongly affecting the accuracy, or more generally to navigate the trade-offs between performance and fairness metrics." This point is similar to the one we're making in the next question.

Reply:
Rephrased.